scripts/lywsd02

```diff
@@ -1,6 +1,7 @@
 #!python
 
 import argparse
+import datetime
```

Comment:
`from datetime import datetime` will be better

Reply:
And replace `client.time = datetime.datetime.now()` with `client.time = datetime.now()`